scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
+// Meter
+// ==========================================================================
+meter {
+	-webkit-appearance: none;
+	width: 550px;
+	height: 25px;
+}
+
+.meter::-webkit-meter-bar, .multiColor::-webkit-meter-bar, .stripMeter::-webkit-meter-bar {
```

Comment:
All of your meter styling is for webkit browsers only.  This means in Firefox it is unstyled.